.husky/pre-commit

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
+. "$(dirname "$0")/_/husky.sh"
+
+npx lint-staged
```

Comment:
used their [migrate cli](https://github.com/typicode/husky-4-to-7) to generate this file.

Reply:
I learned that one can add `exit 1` at the end of this file to abort commit when testing it.